name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/IBFlexStatementExtractor.java

```diff
@@ -512,6 +512,8 @@ private void setAmount(Element element, Transaction transaction, Double amount,
                 {
                     fxRateToBase = new BigDecimal(1);
                 }
+                // TODO why are we dividing by the inverse rate instead of
+                // multiplying with fxRateToBase?

```

Comment:
Agree. It should be good enough to multiply by fxRateToBase.

I assume this is a "path dependency" - the code was changed multiple time in the past...

Reply:
Thanks, consolidated in added commit.